adoptopenjdk-api-v3-updater/src/main/kotlin/net/adoptopenjdk/api/v3/dataSources/mongo/CachedGithubHtmlClient.kt

```diff
@@ -1,19 +1,18 @@
 package net.adoptopenjdk.api.v3.dataSources.mongo
 
-import kotlinx.coroutines.CoroutineScope
-import kotlinx.coroutines.GlobalScope
-import kotlinx.coroutines.asCoroutineDispatcher
-import kotlinx.coroutines.async
-import kotlinx.coroutines.delay
-import kotlinx.coroutines.launch
+import kotlinx.coroutines.*
```

Comment:
prefer explicit imports

Reply:
is this something we can set in editor config or share the IntelliJ settings? I'll check because I'm sure it will slip through again for others (or me)

Reply:
> is this something we can set in the repo's editor config or share the IntelliJ settings?

Looking at the editorconfig is seems like it _should_ configure IntelliJ correctly

```ini
[*.java]
ij_java_class_count_to_use_import_on_demand = 9999
ij_java_names_count_to_use_import_on_demand = 9999
...
[{*.gradle.kts,*.kts,*.kt}]
ij_kotlin_name_count_to_use_star_import = 99
ij_kotlin_name_count_to_use_star_import_for_members = 99
```

There is this line at the bottom of the `kt` section, however. (IntelliJ marks it as invalid though so idk).
```ini
disabled_rules=no-wildcard-imports,...
```

Are you sure the editorconfig plugin is installed and set to override formatting settings for the project in IntelliJ?

Reply:
thanks, good spot

IntelliJ looks like it's set up but the `.editorconfig` doesn't appear to be applied to the project for me - I've set them explicitly anyway in IntelliJ's config for now

there may be some setup here I'm missing though, feels like some notes on this would be a useful addition to the contributing guide but I'm not sure exactly what those notes would be as I've not quite got it working